src/stock-operations/stock-operations-dialog/stock-operations-dialog.component.tsx

```diff
@@ -43,7 +43,7 @@ const StockOperationDialog: React.FC<StockOperationDialogProps> = ({ title, requ
       case 'complete':
         actionName = 'COMPLETE';
         break;
-      case 'completedispatch':
+      case 'complete dispatch':
```

Comment:
We should have a central place we can store this status to make the re-useable and easy to track